src/screens/Tank/SettleBalanceConfirm/SettleBalanceConfirm.js

```diff
@@ -162,11 +162,11 @@ class SettleBalanceConfirm extends React.Component<Props, State> {
 
     const reviewData = [
       {
-        label: 'Assets to settle',
+        label: t('transactions.label.assetsToSettle'),
         valueArray: txToSettle,
       },
       {
-        label: 'Transaction fee',
+        label: t('transactions.label.transactionFee'),
```

Comment:
Looks intentional, but just in case, this string is set to "Est. Network Fee".

Reply:
Yeah, I've unified those :)